### PR TITLE
User registration

### PR DESCRIPTION
**Pull request by Todd-Estes**
**User Story 10 Partially Complete**

-Created Users spec folder with new_spec.rb
-Created Users Controller and  view folder with edit view file

**Next Steps**
-Build User table and User class
-Create session for logged in user
-Ensure registering User has _unique_ email and satisfies all new User requirements
-Complete stories 10-12 and begin 2-9

### CODE REVIEW

**Comment by ShaunDaneJames** [comment]
LGTM!

**Comment by hopesgit** [comment]
Looks great to me!

File: db/migrate/20201027172335_create_users.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class CreateUsers < ActiveRecord::Migration[5.2]
   def change
     create_table :users do |t|
       t.string :name
-      t.string :address
+      t.string :street_address
```
Comment (GeorgieGirl24) [inline db/migrate/20201027172335_create_users.rb:5]: Good job on fixing this migration

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -5,7 +5,10 @@ def new
 
   def create
     @user = User.new(user_params)
-    if !User.where(email_address: @user.email_address).empty?
```
Comment (GeorgieGirl24) [inline app/controllers/users_controller.rb]: Good job on figuring out this logic

**Comment by Todd-Estes** [comment]
Let's get the money!

**Review by GeorgieGirl24 — APPROVED** [review]

**Pull request merged**